api/api/spec/spec.yaml

```diff
@@ -1182,192 +1368,254 @@ components:
           type: object
           properties:
             codename:
+              description: OS version codename.
               type: string
             major:
+              description: Major release version.
               type: string
             minor:
+              description: Minor release version
               type: string
             name:
+              description: OS name.
               type: string
             platform:
+              description: OS platform name.
               type: string
             version:
+              description: Full version name.
               type: string
         scan:
-          type: object
-          properties:
-            id:
-              type: integer
-            time:
-              type: string
-              format: date-time
+          $ref: '#/components/schemas/ScanIdTime'
         release:
+          description: Release name.
           type: string
         architecture:
+          description: OS arquitecture.
           type: string
         version:
+          description: Release version.
           type: string
         sysname:
+          description: System name.
           type: string
         hostname:
+          description: Machine's hostname.
           type: string
         agent_id:
-          type: string
-          minLength: 3
+          $ref: '#/components/schemas/AgentID'
 
     SyscollectorPackages:
       type: object
       properties:
         scan:
-          type: object
-          properties:
-            id:
-              type: integer
-            time:
-              type: string
-              format: date-time
+          $ref: '#/components/schemas/ScanIdTime'
         name:
+          description: Package name.
           type: string
         section:
+          description: Package section.
           type: string
         priority:
+          description: Package priority.
           type: string
         size:
+          description: Installed package size in bytes.
           type: string
         description:
+          description: Brief package description.
           type: string
         architecture:
+          description: Package architecture.
           type: string
         multiarch:
+          description: Whether the package has multiarchitecture support.
           type: string
+          enum:
+          - allowed
+          - same
+          - foreign
+          -
```

Comment:
Delete this empty enum value

Reply:
Done in 32dd319458a697ee0ea73a95b1f1c80b18d80d70